clash-lib/src/Clash/Core/Evaluator/Semantics.hs

```diff
@@ -167,20 +168,40 @@ evaluateVarWith eval i
     Just x  -> forceLocal x
     Nothing -> pure (VNeu (NeVar i))
    where
+    -- TODO Does this need to change?
```

Comment:
I don't know :-). Do you still need to look at this?

Reply:
I don't think this needs to change, so I'm going to take out the TODO